packages/driver/cypress/e2e/commands/location.cy.js

```diff
@@ -23,12 +23,15 @@ describe('src/cy/commands/location', () => {
       cy.url().should('match', /baz/).and('eq', 'http://localhost:3500/foo/bar/baz.html')
     })
 
-    it('catches thrown errors', () => {
-      cy.stub(Cypress.utils, 'locToString')
-      .onFirstCall().throws(new Error)
-      .onSecondCall().returns('http://localhost:3500/baz.html')
+    it('propagates thrown errors from CDP', (done) => {
+      cy.on('fail', (err) => {
```

Comment:
this might be better off in a vite test, but most of these tests are really functioning as some type of integration test 🤷🏻